src/main/java/com/sigma/cinematicketpro/exception/NoSuchRoleException.java

```diff
@@ -0,0 +1,7 @@
+package com.sigma.cinematicketpro.exception;
+
+public class NoSuchRoleException extends RuntimeException {
```

Comment:
I am pretty sure that there is smth in spring security for this

Reply:
Removed this exception as it turned out to be redundant